Presentation.Web/app/helpers/validation-helper.ts

```diff
@@ -0,0 +1,21 @@
+﻿module Kitos.Helpers {
+    export class ValidationHelper {
+        static checkIfStartDateIsGreaterThanEndDate(startDate: moment.Moment, endDate: moment.Moment, notify): boolean {
+            if (startDate >= endDate) {
+                notify.addErrorMessage("Den indtastede slutdato er før startdatoen.");
```

Comment:
"slutdato" og "startdatoen" should both be parameters since they are named differently depending on if it is used in it-system usage or contract